<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewForm</class>
 <widget class="QWidget" name="ViewForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EOS Trigger</string>
  </property>
  <property name="windowIcon">
   <iconset resource="EOSTrigger.qrc">
    <normaloff>:/EOSTrigger_resources/main_icon</normaloff>:/EOSTrigger_resources/main_icon</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>271</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Actions</string>
    </attribute>
    <widget class="QLabel" name="exit_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Exit</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="EOSTrigger.qrc">:/EOSTrigger_resources/application-exit.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="start_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>32</width>
        <height>32</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Start capture</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="EOSTrigger.qrc">:/EOSTrigger_resources/media-playback-start.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="time_remaining_lcdNumber">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>Time remaining</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Outline</enum>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="iso_comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>14</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ISO</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>54</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Delay</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="timing_spinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>3600</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>300</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="loop_checkBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>52</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Loop</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="raw_radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>114</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Raw</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="dark_radioButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>114</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Dark</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="prefix_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>113</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>94</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Prefix (M31,...)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QRadioButton" name="dsusb_radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Use DSUSB</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="mirror_lock_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>95</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Mirror lock</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="usbuart_radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Use USB uart</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="port_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>113</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>/dev/ttyUSB0</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>About</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>221</height>
      </rect>
     </property>
     <property name="plainText">
      <string>EOSTrigger

Simple tool for EOS camera in Bulb mode.

Matmook - Matthieu Barreteau
matthieu@barreteau.org
</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="EOSTrigger.qrc"/>
 </resources>
 <connections/>
</ui>
